docker-images/artifacts/kafka-thirdparty-libs/3.0.x/pom.xml

```diff
@@ -340,11 +340,35 @@
             <groupId>com.fasterxml.jackson.dataformat</groupId>
             <artifactId>jackson-dataformat-yaml</artifactId>
             <version>2.12.3</version>
```

Comment:
@scholzj  - I think this should be updated to `2.12.6` as the version 3.0.1 uses `2.12.6` version. 

Ref: https://cwiki.apache.org/confluence/display/KAFKA/Release+Plan+3.0.1

Reply:
We still support 3.0.0. So it is not so easy.